internal/services/network/subnet_data_source.go

```diff
@@ -83,6 +84,12 @@ func dataSourceSubnet() *pluginsdk.Resource {
 	}
 
 	if !features.FourPointOhBeta() {
+		resource.Schema["private_endpoint_network_policies_enabled"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeBool,
+			Computed:   true,
+			Deprecated: "This is deprecated in favor of `private_endpoint_network_policies`",
```

Comment:
Can you please specify that the property will be removed in v4.0 of the provider 
```suggestion
			Deprecated: "This property has been superseded by `private_endpoint_network_policies` and will be removed in v4.0 of the AzureRM Provider.",
```

Reply:
Updated